cypress/e2e/CMS/$Biography.feature

```diff
@@ -0,0 +1,174 @@
+
+    Scenario: Edit and republish biography content type
+        Given user is navigating to "/user/login"
+        When user enters credentials
+        And user clicks "Log in" button
+        Then user is logged in and the user name "admin" is displayed in the toolbar
+        And the tool bar appears at the top
+        When user clicks on "Content" tab
+        And user clicks on title "Full Name" from the list of content
+        And user clicks on the tool bar status green button "Published"
+        And user clicks "View in edit form" button from other actions
+        And user clears out "Pretty URL" field
+        And user clears out "Card Title" field
+        And user clears out "Meta Description" field
+        And user clears out "Full Name" field
+        And user clears out "Feature Card Description" field
+        And user clears out "Title (value 1)" field
+        And user clears out "Title (value 2)" field
+        And user clears out "Organization 2 Name" field
+        And user clears out "Organization 2 URL" field
+        And user fills out the following fields
+            | fieldLabel               | value                                               | field_name                     |
+            | Pretty URL               | biography-edited                                    | field_pretty_url               |
+            | Full Name                | Full Name Edited                                    | title[0][value]                |
+            | Card Title               | Automated Test Biography - Card Title Edited        | field_card_title               |
+            | Meta Description         | Automated Test Biography - Meta Description Edited  | field_page_description         |
+            | Feature Card Description | Automated Test Biography - Feature Card Desc Edited | field_feature_card_description |
+            | Title (value 1)          | Title 1 Edited                                      | field_title[0][value]          |
+            | Title (value 2)          | Title 2 Edited                                      | field_title[1][value]          |
+            | Organization 2 Name      | DCEG Edited                                         | field_org_name_2               |
+            | Organization 2 URL       | https://dceg.cancer.gov/edited/                     | field_org_url_2                |
+        And user selects "NCI Frederick Campus" from "Campus" dropdown
+        And user selects "Display" from "Show Press Information for Journalists" dropdown
+        And user removes the Biography Image
+        And user selects 2 Biography Image from the list of images
+        And user remembers the source of selected biography image for further verification
+        And user selects "Editing" from "Change to" dropdown
+        When user saves the content page
+        And user clicks on the tool bar status red button "Editing"
```

Comment:
change this to 
 And user clicks on the tool bar status green button "Editing"

( now I know the color of button is actually red, but we already have a step implemented for this, no need to duplicate)